ui/src/app/utils/leo-runtime-initializer.spec.tsx

```diff
@@ -7,7 +7,7 @@ import {Runtime} from 'generated/fetch';
 import {RuntimeStatus} from 'generated/fetch';
 import {RuntimeApi} from 'generated/fetch/api';
 import SpyInstance = jest.SpyInstance;
```

Comment:
No, I wasn't able to combine two imports.
However, I can do this so that we don't need `import SpyInstance = jest.SpyInstance;`

```
let mockGetRuntime: jest.SpyInstance;
let mockCreateRuntime: jest.SpyInstance;
let mockDeleteRuntime: jest.SpyInstance;
let mockStartRuntime: jest.SpyInstance;
```
Let me know if you like me to make this change.